src/ui/screenshot_menu.css

```diff
@@ -41,43 +205,284 @@
+  padding: 0.5rem 0.75rem;
+  font-size: 0.813rem;
+  font-weight: 590;
+  color: var(--gray800);
+  margin: 0 0 0 0.25rem;
+}
+.metacell-theme-screenshot-footer-container .neuroglancer-screenshot-button:disabled {
+  display: none;
+}
+
+.neuroglancer-screenshot-tooltip {
+  position: relative;
+  cursor: pointer;
+}
+
+.neuroglancer-screenshot-tooltip::after {
+  content: attr(data-tooltip);
+  position: absolute;
+  left: 50%;
+  top: 100%; /* Place the tooltip above the element */
+  left: 0;
+  transform: translate(-50%, 0%);
+  text-indent: 0;
+  color: white;
+  padding: 0.5rem 0.75rem;
+  margin-top: 0.375rem;
+  border-radius: 5px;
+  height: auto;
+  width: 20rem;
+  line-height: 1rem;
+  font-size: 0.75rem;
+  font-weight: 590;
+  border-radius: 8px;
+  background: var(--gray600);
   text-align: center;
-  padding: 0.1em;
-  border: 0.1em solid black;
-  width: 1.1em;
-  height: 1.1em;
-  border-radius: 100%;
+  opacity: 0;
+  visibility: hidden;
+  z-index: 9999;
+  transition: opacity 0.3s ease;
+  pointer-events: none; /* Prevent the tooltip from blocking interaction */
+}
+.neuroglancer-screenshot-tooltip::before {
+  content: "";
+  position: absolute;
+  top: 100%; /* Adjust this to move the arrow closer to the tooltip */
+  left: 50%;
+  margin-top: -0.375rem;
+  transform: translateX(-50%);
+  border-width: 0.375rem;
+  border-style: solid;
+  border-color: transparent transparent var(--gray600) transparent; /* Arrow pointing upwards */
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.3s ease;
+  pointer-events: none;
+}
+
+.neuroglancer-screenshot-tooltip:hover::after,
+.neuroglancer-screenshot-tooltip:hover::before {
+  opacity: 1;
+  visibility: visible;
+}
+
+.metacell-theme-copy-icon {
+  position: relative;
+  cursor: pointer;
+}
+
+
+.metacell-theme-copy-icon::after {
+  content: attr(data-tooltip);
+  position: absolute;
+  left: 50%;
+  top: 100%; /* Place the tooltip above the element */
+  left: 0;
+  transform: translate(-50%, 0%);
+  text-indent: 0;
+  color: white;
+  padding: 0.5rem 0.75rem;
+  margin-top: 0.375rem;
+  border-radius: 5px;
+  height: auto;
+  width: 8.125rem;
+  line-height: 1rem;
+  font-size: 0.75rem;
+  font-weight: 590;
+  border-radius: 8px;
+  background: var(--gray600);
+  text-align: center;
+  opacity: 0;
+  visibility: hidden;
+  z-index: 9999;
+  transition: opacity 0.3s ease;
+  pointer-events: none; /* Prevent the tooltip from blocking interaction */
+}
+.metacell-theme-copy-icon::before {
+  content: "";
+  position: absolute;
+  top: 100%; /* Adjust this to move the arrow closer to the tooltip */
+  left: 50%;
+  margin-top: -0.375rem;
+  transform: translateX(-50%);
+  border-width: 0.375rem;
+  border-style: solid;
+  border-color: transparent transparent var(--gray600) transparent; /* Arrow pointing upwards */
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.3s ease;
+  pointer-events: none;
+}
+
+.metacell-theme-copy-icon:hover::after,
```

Comment:
Thank you @vidhya-metacell! This looks great at a first glance. I'll take a detailed look soon, but one top-level question:

Would you be ok if we just replace all occurences of `metacell` by `screenshot`? I think that will make it easier when merging back into the google/neuroglancer main branch